storage/templates/provisioner.yaml

```diff
@@ -101,3 +123,32 @@ metadata:
 type: "kubernetes.io/rbd"
 data:
   key: QVFDMkI0SmE2MDgzT3hBQTdOR0dpb0xpR1lqOHlJTFpkYUI1T0E9PQo=
+---
+kind: PersistentVolumeClaim
```

Comment:
That's the wrong location. Let's create an "example" folder and put the test yaml there.